lib/matplotlib/container.py

```diff
@@ -18,7 +18,7 @@ def __repr__(self):
     def __new__(cls, *kl, **kwargs):
         return tuple.__new__(cls, kl[0])
 
-    def __init__(self, kl, label=None):
+    def __init__(self, kl, label=None, lorder=''):
```

Comment:
Hmm. I can think of a couple of options. In no particular order:

### Use "last"
I *think* this communicates what the default value means and preserves your intent and the control logic.
```python
def __init__(self, kl, label=None, lorder='last'):
    # ...
```

### Replace `None` immediately
```python
def __init__(self, kl, label=None, lorder=None):
    lorder  = '' or lorder
    # ...
```
That's all I can think of for now.